picard/util/__init__.py

```diff
@@ -175,20 +179,45 @@ def decode_filename(filename):
         return filename.decode(_io_encoding)
 
 
+def system_supports_long_paths():
+    """Detects long path support.
+
+    On Windows returns True, only if long path support is enabled in the registry (Windows 10 1607 or later).
+    All other systems return always True.
+    """
+    if not IS_WIN:
+        return True
+    else:
```

Comment:
We could drop this `else` since the other branch returns, in order to reduce indentation level for the rest of the code.